utilities/ordered-set.metta

```diff
@@ -142,12 +142,15 @@
 ;; Returns: (OS Number) → Union of both sets
 ;; Example: (OS.union (ConsOS 0 (ConsOS 1 NilOS)) (ConsOS 2 NilOS)) → (ConsOS 0 (ConsOS 1 (ConsOS 2 NilOS)))
 (: OS.union (-> (OS Number) (OS Number) (OS Number)))
-(: OS.union (-> (OS Number) (OS Number) (OS Number)))
-(= (OS.union NilOS $set2) $set2)
-(= (OS.union (ConsOS $head $tail) $set2)
-   (chain (OS.insert genericCompare $head $set2) $newSet2
-     (OS.union $tail $newSet2)
-   )
+(= (OS.union $set1 $set2)
```

Comment:
I can't stop but wonder, since this change was previously implemented using normal function definitions instead of a `unify` function, it was a result of incorrect merge conflict handling. The `unify` isn't stable and causes errors. If the change wasn't intentional can you make sure the previous change is used instead?